trezorlib/tests/device_tests/test_multisig_change.py

```diff
@@ -135,7 +136,16 @@ def _responses(self, inp1, inp2, change=0):
             proto.TxRequest(
                 request_type=proto.RequestType.TXINPUT,
                 details=proto.TxRequestDetailsType(request_index=0),
-            ),
+            )
+        ]
+        if (
+            TREZOR_VERSION != 1
+        ):  # trezor 1 does not have UnknownDerivationPath implemented
```

Comment:
please move the comment to the next line, then black will not make this particular mess :)